File: minning.js
const csv = require("csv-parser");
const fs = require("fs");

const c1 = [10],
  c2 = [],
  c3 = [],
  c4 = [],
  c5 = [],
  c6 = [],
  match = [];

const values = ["5", "05", "10", "60"];
let five = 0,
  zeroFive = 0,
  ten = 0,
  sixtieth = 0;

fs.createReadStream("./megasena.csv")
  .pipe(csv())
  .on("data", (row) => {
    c1.push(row.c1);
    c2.push(row.c2);
    c3.push(row.c3);
    c4.push(row.c4);
    c5.push(row.c5);
    c6.push(row.c6);
  })
  .on("end", () => {
    function isMatch(value) {
      switch (value) {
        case "5":
          five++;
          break;
        case "05":
          zeroFive++;
          break;
        case "10":
          ten++;
          break;
        case "60":
          sixtieth++;
          break;
        default:
          break;
      }
    }

    const arrays = [c1, c2, c3, c4, c5, c6];

    arrays.forEach((array) => {
      var filtered = array.filter(isMatch);
    });

    console.log(
      `Quantidade de "05": ${zeroFive}\nQuantidade de  "5": ${five}\nQuantidade de "10": ${ten}\nQuantidade de "60": ${sixtieth}`
    );
  });

File: minning.py
import pandas as pd
import csv

megasena = pd.read_csv('./megasena.csv')

c1 = megasena["c1"].to_list()
c2 = megasena["c2"].to_list()
c3 = megasena["c3"].to_list()
c4 = megasena["c4"].to_list()
c5 = megasena["c5"].to_list()
c6 = megasena["c6"].to_list()

zeroFive =0
five = 0
ten = 0
sixtieth = 0

controlArray = [c1,c2,c3,c4,c5,c6]

for array in controlArray:
    for number in array:
      if (number == "05"):
        print(number)
        zeroFive += 1
      if (number == 5):
        five += 1
      if (number == 10):
        ten += 1
      if (number == 60):
        sixtieth += 1
print('05 apareceu %d vezes\n5 apareceu %d vezes\n10 apareceu %d vezes\n60 apareceu %d vezes' % (zeroFive, five, ten, sixtieth))
          
          



    
    
    



